lib/srv/desktop/tdp/proto.go

```diff
@@ -1460,3 +1470,14 @@ func writeUint64(b *bytes.Buffer, v uint64) {
 	b.WriteByte(byte(v >> 8))
 	b.WriteByte(byte(v))
 }
+
+// tdpMaxNotificationMessageLength is somewhat arbitrary, as it is only sent *to*
+// the browser (Teleport never receives this message, so won't be decoding it)
+const tdpMaxNotificationMessageLength = 10240
+
+// tdpMaxPathLength is currently just an arbitrary value, more research is required
+// (see https://github.com/gravitational/teleport/issues/14950)
+const tdpMaxPathLength = 10240
+
+const maxClipboardDataLength = 1024 * 1024    // 1MB
```

Comment:
Is this MB or KB?

Reply:
1MB == 2^20 bytes == `0x100000` == (1KB)^2 == 1024 * 1024 bytes

Reply:
Yes... 🤦